lms/templates/email/preferences.html.jinja2

```diff
@@ -0,0 +1,33 @@
+{% extends 'templates/base.plain.html.jinja2' %}
+
+{% block content %}
+{% for message in request.session.pop_flash() %}
+  <p><b>{{ message }}</b></p>
+{% endfor %}
+
+<form action="" method="post">
+  <fieldset>
+    <legend>Receive email notifications when your students annotate. Select
+    the days you'd like your emails:</legend>
+
+    <ul style="list-style-type:none;">
+      <li><input type="checkbox" id="sunday" name="instructor_email_digests.days.7" {% if preferences["instructor_email_digests.days.7"] %}checked{% endif %} />
```

Comment:
Same as in the service, these dictionary keys of the model leak all the way to the template.

IMO it would be nicer to hide those.